src/Service.GraphQLBuilder/Mutations/MutationBuilder.cs

```diff
@@ -20,7 +20,8 @@ public static class MutationBuilder
         /// The item field's metadata is of type OperationEntityInput
         /// i.e. CreateBookInput
         /// </summary>
-        public const string INPUT_ARGUMENT_NAME = "item";
+        public const string ITEM_INPUT_ARGUMENT_NAME = "item";
+        public const string ARRAY_INPUT_ARGUMENT_NAME = "items";
```

Comment:
Nope. I'm gonna remove the ones ins `CreateMutationBuilder` since the constants are generic and should be reused across other mutation types (update/delete) as well.